src/OrbitGl/TriangleToggle.h

```diff
@@ -51,7 +53,7 @@ class TriangleToggle : public orbit_gl::CaptureViewElement,
   // We require explicit knowledge about the parent.
   Track* track_;
 
-  float height_;
+  float height_ = 20;
```

Comment:
Why this wasn't needed and now is needed?

Reply:
Maybe I shouldn't have added this to the PR... it was like this before, I've just seen the warning that this was never initialized, so I added it.